airbyte-workers/src/main/java/io/airbyte/workers/process/ProcessBuilderFactory.java

```diff
@@ -30,19 +30,27 @@
 
 public interface ProcessBuilderFactory {
 
-  ProcessBuilder create(String jobId, int attempt, final Path jobPath, final String imageName, final String... args) throws WorkerException;
+  ProcessBuilder create(String jobId, int attempt, final Path jobPath, final String imageName, final String entrypoint, final String... args)
+      throws WorkerException;
 
-  default ProcessBuilder create(long jobId, int attempt, final Path jobPath, final String imageName, final String... args) throws WorkerException {
-    return create(String.valueOf(jobId), attempt, jobPath, imageName, args);
+  default ProcessBuilder create(long jobId, int attempt, final Path jobPath, final String imageName, final String entrypoint, final String... args)
+      throws WorkerException {
+    return create(String.valueOf(jobId), attempt, jobPath, imageName, entrypoint, args);
   }
 
-  default ProcessBuilder create(String jobId, int attempt, final Path jobPath, final String imageName, final List<String> args)
+  default ProcessBuilder create(String jobId,
+                                int attempt,
+                                final Path jobPath,
+                                final String imageName,
+                                final String entrypoint,
+                                final List<String> args)
       throws WorkerException {
-    return create(jobId, attempt, jobPath, imageName, args.toArray(new String[0]));
+    return create(jobId, attempt, jobPath, imageName, entrypoint, args.toArray(new String[0]));
```

Comment:
I might be missing something - I don't actually see where this is used. I've looked in all usages of the create method so far. What's the purpose for adding this?